src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -123,6 +123,10 @@ module sonic-vlan {
 					default disable;
 				}
 
+				leaf mac_addr {
```

Comment:
this is an optional field.

Reply:
This doesn't have the `mandatory true;` attribute and also doesn't have a default value .... so it is therefore optional as far as I am aware.

What am I missing here?